feature-libs/cart/base/styles/components/_item-counter.scss

```diff
@@ -79,6 +80,14 @@
 
     &:disabled {
```

Comment:
As far as I understand this selector will be useless after your changes become permanent. let's leave a TODO comment to clean those up after, something like:
`// TODO: The styles for ":disabled" selector should be deleted when "a11yItemCounterFocus" feature flag has been removed`